connexion/operation.py

```diff
@@ -169,28 +168,12 @@ def __init__(self, method, path, path_parameters, operation, app_produces,
         self.endpoint_name = flaskify_endpoint(self.operation_id)
         self.__undecorated_function = resolution.function
 
-        for param in self.parameters:
-            if param['in'] == 'body' and 'default' in param:
-                self.default_body = param
-                break
-        else:
-            self.default_body = None
-
         self.validate_defaults()
 
     def validate_defaults(self):
         for param in self.parameters:
             try:
-                if param['in'] == 'body' and 'default' in param:
```

Comment:
This validation is handled by the "swagger_spec_validation" pkg, we do not need to do it anymore.